src/handlers.jl

```diff
@@ -134,8 +140,8 @@ function complete_request(socket, msg)
     end
 
     codestart = find_parsestart(code, cursorpos)
-    comps, positions = Base.REPLCompletions.completions(code[codestart:end], cursorpos-codestart+1)
-    positions += codestart-1
+    comps, positions = REPLCompletions.completions(code[codestart:end], cursorpos-codestart+1)
+    positions = positions .+ codestart-1
```

Comment:
Nevermind, it is this fix that is missing a `.` on the `-` and the range turned into an array. Of course last works for an empty range.